### ISSUE TITLE
error: ios16 Location.getLocation 

### ISSUE THREAD

**Issue by jieun8206** [Lyokone/flutterlocation#782]
ios 16.

requested "location.getLocation()"

console is
This method can cause UI unresponsiveness if invoked on the main thread.
'-locationManagerDidChangeAuthorization:' callback and checking 'authroizationStatus' first.

**Comment by cosmosdesigner** [comment]
Hello, any news @Lyokone ? This bug is causing some erros on apple center. Thank you for your time.

**Comment by luvishq** [comment]
any updates ?

**Comment by elninho22** [comment]
+1
we have a forecast for a solution the impact is gigantic.

**Comment by xunreal75** [comment]
Please use another permissions package to check location rights like https://pub.dev/packages/permission_handler 

The package seems dead at the moment. No answers by developer or contributors.

**Comment by elninho22** [comment]
> Use outro pacote de permissões para verificar os direitos de localização, como https://pub.dev/packages/permission_handler
> 
> O pacote parece morto no momento. Nenhuma resposta do desenvolvedor ou contribuidor.

Yes, I already use this package. but I still have problems and not getting the location of the user.
